addons/web/static/lib/nvd3/nv.d3.css

```diff
@@ -153,7 +153,7 @@ svg {
   /* Trying to get SVG to act like a greedy block in all browsers */
   display: block;
   width:100%;
-  height:100%;
+  height:auto;
```

Comment:
the words "certainly a bad idea" aren't strong enouth.  This is definitely bad.  Don't do this.